### PR TITLE
FIX: Updating with working versions

### PR DESCRIPTION
**Pull request by maxpastor**
When trying to use this doc I used this code out of the box. It didn't work. 
I am suggesting these modifications, after having tested them. 

Why am I not using the very last versions appearing in the reference?
Newer versions are not compatible with the Sagemaker SDK versions provided in the "Get started" page. These versions are the latest versions to be working with that SM SDK version. 

Why am I using the "get started" pip install versions ? 

Because they work with what I suggested and newer versions seem to be bringing issues with Datasets. 
I had the same issue as this one, using new versions : https://github.com/huggingface/datasets/issues/5150

### CODE REVIEW

File: docs/sagemaker/train.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ from sagemaker.huggingface import HuggingFace
 # hyperparameters which are passed to the training job
 hyperparameters={'epochs': 1,
                  'per_device_train_batch_size': 32,
-                 'model_name_or_path': 'distilbert-base-uncased'
+                 'model_name': 'distilbert-base-uncased'
```
Comment (philschmid) [inline docs/sagemaker/train.md:123]: same as aboveI

**Review by philschmid — COMMENTED** [review]
Added some comments. not sure if those changes are needed.

File: docs/sagemaker/train.md [hunk 2]
```diff
@@ -83,7 +83,7 @@ if __name__ == "__main__":
     # hyperparameters sent by the client are passed as command-line arguments to the script
     parser.add_argument("--epochs", type=int, default=3)
     parser.add_argument("--per_device_train_batch_size", type=int, default=32)
-    parser.add_argument("--model_name_or_path", type=str)
+    parser.add_argument("--model_name", type=str)
```
[the remark this answers is not in the document]
Comment (maxpastor) [reply docs/sagemaker/train.md:86]: On line 95 the script is referenced. Still, if the intent of the page is to be more generic, then I would agree with you and discard these 2 modifications.

**Comment by osanseviero** [comment]
HI there! This PR has had no activity in 1 year. I'll close it but feel free to reopen if it's still relevant

**Pull request closed without merging**